dts/bindings/spi/snps,designware-spi.yaml

```diff
@@ -32,4 +32,10 @@ properties:
       category: required
       description: required interrupts
       generation: define
+
+    interrupt-masq:
```

Comment:
I dont see this used anywhere in this patch set.  Can we drop it for now, I'm still confused what this is suppose to be used for.

Reply:
How come you don't see it? Is github messing up things or?

See "dts/x86: Add the 2 master SPI port on curie SoC"
and "boards/x86: Enable usage of SPI definition through dts"

Reply:
I see it now, I wasn't tracing the #define's enough.